tests/init.spec.js

```diff
@@ -4,8 +4,7 @@ const {
   existsSync, mkdtempSync, readFileSync, rmdirSync,
 } = require('fs');
 const { parse, resolve } = require('path');
-// Path to cli relative to tmp working directory.
-const cli = '../../../cli.js';
+const { cli } = require('./lib/cli');
```

Comment:
This declaration got a bit involved to support running tests in Windows and I felt it should have a comment explaining what we did. So I moved it to its own file and now any other test suite (`*.spec.js` file) can require the `tests/lib/cli.js` file to get the command without having to re-declare everything.